ml/src/main/scala/ai/h2o/sparkling/ml/utils/EstimatorCommonUtils.scala

```diff
@@ -82,4 +83,11 @@ trait EstimatorCommonUtils extends RestCommunication {
     } while (H2OModel.modelExists(replacement))
     replacement
   }
+
+  private def printWarnings(messages: Array[ValidationMessageV3]): Unit = {
+    messages
+      .filter(_.message_type == "WARN")
+      .map(warn => s"${warn.message} (field name: ${warn.field_name})")
+      .foreach(logWarning(_))
```

Comment:
Shouldn't the warnings go to stdout to be visible in jupyter or databricks notebooks?

Reply:
done, I noticed h2o-3 python client uses "warnings.warn" which prints to stderr so I used that, please let me know if that's ok, I'm fine with changing that to stdout

I think pysparkling calls that code and the messages sent to stdout/stderr are printed in jupyter notebook

Reply:
If H2O python client uses `stderr`, your solution is correct!

> I think pysparkling calls that code and the messages sent to stdout/stderr are printed in jupyter notebook
yes, pysparkling calls the scala code via py4j, but it's better to check that it works since, python code and scala code run in different processes.

Reply:
I decided to add "WARN:" before the message

a little proof that it works:
<img width="1344" alt="Screenshot 2021-12-09 at 17 09 55" src="https://user-images.githubusercontent.com/8573352/145486860-97364f49-6852-4acf-9007-6b1aa0e2f621.png">